src/RichText/Toolbar/actions.ts

```diff
@@ -143,20 +143,22 @@ export const DEFAULT_TOOLBAR_ITEMS: ToolbarItem[] = [
   },
   {
     onPress:
-      ({ editor }) =>
+      ({ editor, editorState }) =>
       () =>
-        editor.sink(),
+        editorState.canSink ? editor.sink() : editor.sinkTaskListItem(),
     active: () => false,
-    disabled: ({ editorState }) => !editorState.canSink,
+    disabled: ({ editorState }) =>
+      !editorState.canSink && !editorState.canSinkTaskListItem,
```

Comment:
nit: can you comment on the code explaining that logic? we merged both sink/lift (list & checklist) functionality